.stickler.yml

```diff
@@ -0,0 +1,16 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './.eslintrc.json'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+      - 'webpack.config.js' 
```

Comment:
Please do not modify the provided configuration files: https://github.com/microverseinc/linters-config/blob/master/javascript/.stickler.yml.